docs/core/compatibility/globalization.md

```diff
@@ -9,11 +9,16 @@ The following breaking changes are documented on this page:
 
 | Breaking change | Version introduced |
 | - | :-: |
+| [Globalization APIs use ICU libraries on Windows](#globalization-apis-use-icu-libraries-on-windows) | 5.0 |
```

Comment:
Do we usually mention the `Preview` that it happened?

Reply:
Not usually in this navigation table.